src/app/grid/grid-select/grid-select-sample.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="grid__wrapper">
+    <div class="sample__header">
+        <igx-grid #grid1 [data]="data" [primaryKey]="'name'" width="780px" height="600px">
+            <igx-column field="name" header="Character name" [editable]="true" width="25%">
+            </igx-column>
+            <igx-column field="race" header="Race" [editable]="true" [filterable]="false" width="20%">
+                <ng-template igxCellEditor let-cell="cell" let-value>
+                    <igx-select [(ngModel)]="cell.editValue" width="100%" [igxFocus]="true">
+                        <igx-select-item *ngFor="let race of races" [value]="race">
```

Comment:
It will be more readable if the igx-select-item **closing tags** are on new lines